kokoro/linux/dockerfile/test/ruby/Dockerfile

```diff
@@ -38,6 +38,7 @@ RUN /bin/bash -l -c "rvm install 3.0.0"
 RUN /bin/bash -l -c "rvm install 3.1.0"
 RUN /bin/bash -l -c "rvm install jruby-9.2.20.1"
 RUN /bin/bash -l -c "rvm install jruby-9.3.3.0"
```

Comment:
Is this one used elsewhere?

Reply:
It is used currently, and it wasn't clear to me if I removed it immediately if it would be disruptive for other PRs.